### PR TITLE
Work and about section

### PR DESCRIPTION
**Pull request by Naledi-Dikgale**
**The Mobile Portfolio Project features:**

In this pull request, the following was done:

- created an index.html file that includes the body section of mobile portfolio project
- created an index.css file that styles the buttons , text and images
- added a test.md that is ignored by git

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,232 @@
...
+      
+      </div>
+  </div>
+
+</div>
+</section>
+
+<section class="about">
+      <div class="about-intro-text">
+          <p class="project-name h-text">About Myself</p>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] Could you please make this a title tag as in `hx` - `2 - 6`? This seems more like a heading for the `p` content that follows and will make your code more semantically correct. :wink:

File: index.html [hunk 2]
```diff
@@ -1,13 +1,232 @@
...
+                <h4 class="about-heading-small">Frameworks</h4>
+                <i class="fa-solid fa-angle-right"></i>
+            </div>
+        </li>
+        <hr>
+        <li class="about-skill-container">
+            <div class="d-flex about-skill-heading">
+                <h4 class="about-heading-small">Skills</h4>
+                <i class="fa-solid fa-angle-right"></i>
+            </div>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] Well done with this implementation :clap: It's only more semantic to use `ul`/`li` relationship to itemize your Language and Skils items. Kindly use an unordered list `ul`, `li` to nest each of your `Language`, `Frameworks` and `Skills` section :muscle:

File: index.html [hunk 3]
```diff
@@ -1,13 +1,232 @@
...
+            <ul class="tech-list">
+              <li class="project-tech">html</li>
+              <li class="project-tech">css</li>
+              <li class="project-tech">javascript</li>
+            </ul>
+              <button class="outlined-button">See Project</button>
+          
+          </div>
+      </div>
+
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] For each of your project cards, kindly check to make sure that your spacing units (whether `padding`, `margin` or `gap`) match the figma design. Also, `font-sizes`, `background-color`, and `colors`. It is important that you take those specific properties from the template given you. Kindly treat it as your clients requirement which you should try as much as possible to deliver.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

- [ ] Also, for all you buttons you seem to forget to add some interactions like `hover` and `active` states. Kindly add them as specified by the Figma design. Here is a copy of the design palette :point_down: 

image

File: index.html [hunk 4]
```diff
@@ -1,13 +1,232 @@
...
+        <hr>
+        <li class="about-skill-container">
+            <div class="d-flex about-skill-heading">
+                <h4 class="about-heading-small">Skills</h4>
+                <i class="fa-solid fa-angle-right"></i>
+            </div>
+        </li>
+        <hr>
+    </ul>
+    </section>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] Like I pointed on your project cards, the design are great but they missing some tiny bits but important details to make your design really as expected and to show your great knack for smart designs. Some `padding`s are too much, making the design a little unpleasant while fonts like the `About Me` don't have the correct value, Please kindly compare and make them match :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @Naledi-Dikgale :wave: :clap: :muscle: , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Nice implementation :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -1,13 +1,250 @@
...
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
+      </div>
+  </div>
+</section>
+<section class="about">
+      <div class="about-intro-text">
+          <h1 class="project-name h-text1">About <br> My self</h1>
```
Comment (mckent05) [inline index.html]: - To help improve the SEO of your site, it is advisable to use only one `h1` tag on your site. Please, just as the previous reviewers advised please use an `hx` tag between `h2-h6`. Please fix this 🙏

File: index.html [hunk 6]
```diff
@@ -1,13 +1,250 @@
...
+              <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">2015</span></li>
+            </ul>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        <ul class="tech-list">
+          <li class="project-tech">html</li>
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
```
Comment (mckent05) [inline index.html]: - Like the previous reviewer suggested, please implement the hover styles for all your buttons. Kindly add them as specified by the Figma design. Here is a copy of the design palette 👇
Design template:
interactions

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Naledi-Dikgale, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Naledi-Dikgale 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ **OPTIONAL** ] I think it would be better if you remove your ken image logo at the top of you README file to make it more professional

_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**